doc/classes/ConfigFile.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8" ?>
+	<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
This is likely a mistake, this tab should not be added.

Reply:
I thought I removed that tab... I'll fix that in a bit when I get to my computer.